dwellingly/src/App.js

```diff
@@ -81,13 +83,26 @@ export class App extends React.Component {
       <UserContext.Provider value={{ user: { ...this.state.userSession }, login: this.login, logout: this.logout }} >
         <BrowserRouter>
           <div className='App'>
+            <NavMenu />
             <Header />
-
             <Switch>
               <PrivateRoute exact path='/' component={Dashboard} />
-              <Route exact path='/login' component={LoginForm} />
+              <PrivateRoute exact path='/login' component={LoginForm} />
```

Comment:
Login should still be a normal Route since you have to login to get to the PrivateRoutes :)

Terms should also be a normal Route